tests/unit/nupic/encoders/date_test.py

```diff
@@ -57,69 +57,69 @@ def setUp(self):
     dayOfWeekExpected = [0,0,0,1,0,0,0]
 
     # not a weekend, so it should be "False"
-    weekendExpected = [1,1,1,0,0,0]
+    weekendExpected = [1, 0]
 
-    # time of day has radius of 4 hours and w of 5 so each bit = 240/5 min = 48min
-    # 14:55 is minute 14*60 + 55 = 895; 895/48 = bit 18.6
+    # time of day has radius of 4 hours and w of 5 so each bit = 240/5
+    # min = 48min 14:55 is minute 14*60 + 55 = 895; 895/48 = bit 18.6
     # should be 30 bits total (30 * 48 minutes = 24 hours)
-    timeOfDayExpected = [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,0,0,0,0,0,0,0,0,0]
-    self._expected = numpy.array(seasonExpected + dayOfWeekExpected + weekendExpected \
-                          + timeOfDayExpected, dtype=defaultDtype)
+    timeOfDayExpected = (
+      [0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,1,1,1,0,0,0,0,0,0,0,0,0])
+    self._expected = numpy.array(seasonExpected +
+                                 dayOfWeekExpected +
+                                 weekendExpected +
+                                 timeOfDayExpected, dtype=defaultDtype)
 
   def testDateEncoder(self):
-    '''creating date encoder instance'''
-    self.assertEqual(self._e.getDescription(), [("season", 0), ("day of week", 12),
-                                ("weekend", 19), ("time of day", 25)])
-
-    self.assertTrue((self._expected == self._bits).all())
+    """creating date encoder instance"""
+    self.assertSequenceEqual(
+      self._e.getDescription(),
+      [("season", 0),
+       ("day of week", 12),
+       ("weekend", 19), ("time of day", 21)])
+    self.assertTrue(numpy.array_equal(self._expected, self._bits))
 
-    print
-    self._e.pprintHeader()
-    self._e.pprint(self._bits)
-    print
 
   def testMissingValues(self):
-    '''missing values'''
+    """missing values"""
     mvOutput = self._e.encode(SENTINEL_VALUE_FOR_MISSING_DATA)
     self.assertEqual(sum(mvOutput), 0)
 
+
   def testDecoding(self):
-    '''decoding date'''
+    """decoding date"""
     decoded = self._e.decode(self._bits)
 
-    (fieldsDict, fieldNames) = decoded
+    (fieldsDict, _) = decoded
```

Comment:
sometimes it is useful to keep a descriptive name like `_fieldNames`. May not be important here but just to keep in mind that lint will understand it as unused still